cmd/snap-confine/snap-confine.c

```diff
@@ -315,8 +315,8 @@ int main(int argc, char **argv)
 				}
 			}
 			struct snappy_udev udev_s;
-			if (snappy_udev_init(security_tag, &udev_s) == 0)
-				setup_devices_cgroup(security_tag, &udev_s);
+			snappy_udev_init(security_tag, &udev_s);
```

Comment:
Currently snappy_dev_init() returns an int, 0 if devices are assigned and -1 otherwise. This is called in exactly one place; right here and is setup that way for the conditional nature for when we setup the cgroup. Since we want to go unconditional, please adjust snappy_udev_init() to return void and adjust the comment above the function